session-02/exercise.js

```diff
@@ -3,7 +3,16 @@
   For example, for the input ["cat", "hat"], return ["CAT", "HAT"]
 */
 function transformArrayToUpper(listOfStrings) {
-  return;
+  
+  let i = 0;
+  while(i<listOfStrings.length)
+   {let word =listOfStrings[i].toUpper;
```

Comment:
Good idea here, the function is `.toUpperCase()` and make sure to call it using the parentheses.